### ISSUE TITLE
Invalid prop `noDataText` browser console error

### ISSUE THREAD

**Issue by efalayi** [reactioncommerce/reaction#3219]
### Expected behavior
No errors in console when setting is clicked

### Actual behavior
Show `Invalid prop noDataText` in browser console

![dashboard-error](https://user-images.githubusercontent.com/28438068/32290311-fe1c27cc-bf39-11e7-80fd-8f7dcc73c774.gif)

### Steps to reproduce the behavior
- Pull master branch
- Start application and refresh page
- Click on `Accounts` setting
- Observer error in browser console

### Versions
```
Node: 8.3.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.10.0
Reaction: 1.5.1
```

**Comment by impactmass** [comment]
This shows up when I open Accounts, Email, or MarketplaceShops settings panel, likely because they all use SortableTable.

**Comment by efalayi** [comment]
@impactmass yes. I think somewhere along the line, `noDataText` props wasn't defined. `noDataText` is required in SortableTable.

**Comment by zenweasel** [comment]
Closed via #3291
